src/modules/Dashboard/features/Applet/Overview/Overview.utils.tsx

```diff
@@ -0,0 +1,151 @@
+          <StyledFlexTopCenter component="span" sx={{ gap: 1.6 }}>
+            {(participantsCount ?? 0).toLocaleString(i18n.language)}
+
+            <IconButton onClick={commonProps.onPressAddParticipant}>
+              <Svg aria-label={i18n.t('addParticipant')} id="add" width={20} height={20} />
+            </IconButton>
+          </StyledFlexTopCenter>
+        ),
+      },
+      {
+        label: i18n.t('appletOverview.statSubmissions'),
+        icon: 'fileCheck' as const,
+        value: (submissionsCount ?? 0).toLocaleString(i18n.language),
+      },
+    ],
+  };
+}
+
+export function mapResponseToSubmissionsTableProps({
+  submissions = [],
+  submissionsCount = 0,
+}: GetAppletSubmissionsResponse) {
+  return {
+    'data-testid': 'recent-submissions',
+    columns: [
+      { id: 'activity', label: i18n.t('appletOverview.columnActivity') },
+      { id: 'respondent', label: i18n.t('appletOverview.columnRespondent') },
+      { id: 'subject', label: i18n.t('appletOverview.columnSubject') },
+      { id: 'submissionDate', label: i18n.t('appletOverview.columnSubmissionDate') },
+    ],
+    count: submissionsCount ?? submissions?.length,
+    handleRequestSort: () => {},
+    order: 'desc' as const,
+    orderBy: '',
+    rows:
+      submissions.length > 0
+        ? submissions.map(
+            ({
+              activityName,
+              createdAt,
+              sourceSubjectId,
+              sourceNickname,
+              sourceSubjectTag,
+              targetSubjectId,
+              targetNickname,
+              targetSubjectTag,
+            }) => {
+              const createdAtDate = new Date(createdAt);
+
+              return {
+                activity: {
+                  content: () => <StyledMaybeEmpty>{activityName}</StyledMaybeEmpty>,
+                  value: true,
+                },
+                respondent: {
+                  content: () => (
+                    <StyledMaybeEmpty>
+                      <ParticipantSnippet
+                        tag={sourceSubjectTag}
+                        secretId={sourceSubjectId}
+                        nickname={sourceNickname}
+                      />
+                    </StyledMaybeEmpty>
+                  ),
+                  sx: { maxWidth: theme.spacing(31.2) },
+                  value: true,
+                },
+                subject: {
+                  content: () => (
+                    <StyledMaybeEmpty>
+                      {sourceSubjectId === targetSubjectId ? (
+                        i18n.t('appletOverview.labelSelf')
+                      ) : (
+                        <ParticipantSnippet
+                          tag={targetSubjectTag}
+                          secretId={targetSubjectId}
+                          nickname={targetNickname}
+                        />
+                      )}
+                    </StyledMaybeEmpty>
+                  ),
+                  sx: { maxWidth: theme.spacing(31.2) },
```

Comment:
Seems the row heights are collapsing a bit in height without the presence of the icon buttons; to maintain the sense of space here and align with Participant & Manager table styling, a minimum row height might be in order (I guess `<td>`s treat `height` as minimum height, and don't respond to `minHeight` 🤷🏻‍♂️).

I suppose this could also be part of the `DashboardTable` base styling? I noticed it feels a little too tight on `/dashboard/respondents` too.
```suggestion
                  sx: { maxWidth: theme.spacing(31.2), height: theme.spacing(6.4) },
```

Reply:
I went ahead and added it to the `DashboardTable` directly — I wound up having to apply it to the table cell styling specifically, rather than at the row level, but 🤷‍♀️.